### PR TITLE
[swig] fix ConfigDBConnector.db_name

### PR DESCRIPTION
**Pull request by qiluo-msft**
Originally this property is not working.
Add unit test.

### CODE REVIEW

File: common/configdb.h [hunk 1]
```diff
@@ -27,18 +27,49 @@ class ConfigDBConnector_Native : public SonicV2Connector_Native
     virtual std::map<std::string, std::map<std::string, std::map<std::string, std::string>>> get_config();
 
     std::string getKeySeparator() const;
+    std::string getDbName() const;
+
+protected:
+    std::string TABLE_NAME_SEPARATOR = "|";
```
Comment (arlakshm) [inline common/configdb.h:33]: do we need to add property for `TABLE_NAME_SEPARATOR` as well ?

Comment (kcudnik) [reply common/configdb.h:33]: why those 2 members are capita letters like defines ? use common naming convention

Comment (qiluo-msft) [reply common/configdb.h:33]: @kcudnik I am trying to mimic py-swsssdk https://github.com/Azure/sonic-py-swsssdk/blob/9d019b0f330710588f37d112104c563087dc9529/src/swsssdk/configdb.py#L55

Comment (qiluo-msft) [reply common/configdb.h]: @kcudnik Agree with you and fixed.

Comment (qiluo-msft) [reply common/configdb.h]: @arlakshm Agree and fixed.

**Review by arlakshm — APPROVED** [review]
lgtm

**Review by kcudnik — APPROVED** [review]

**Pull request merged**